packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -936,10 +939,12 @@ export class RpcClientCache implements RpcClientInterface {
     data: PackDataParams,
     { block }: RPCOptions = defaultRPCOptions
   ): Promise<{ packed: string; gas: BigNumber | 'unaccounted' | undefined }> {
-    const key = this.formatCacheKey(this.rpcClient.getRpcUrl(), RPCMethodName.PACK_DATA, [
-      block,
-      data,
-    ]);
+    const key = this.formatCacheKey(
+      this.rpcClient.getRpcUrl(),
+      RPCMethodName.PACK_DATA,
+      [block],
+      data
```

Comment:
same comment as above

Reply:
cuz data is a request body not a param this is the formatCacheKey signature ```private formatCacheKey(
    rpcUrl: string,
    rpcMethodName: RPCMethodName,
    rpcMethodParams: RpcMethodParam[],
    rpcMethodData?: object
  )```